reg_tests/update_baseline.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -x
+
+HOMEreg=$1
+test_name=$2
+commit_num=$3
+
+base_dir=$HOMEreg/baseline_data
+base_dir_commit=${base_dir}/$test_name.$commit_num
+
+chmod 755 $base_dir
+
+if [ -d $base_dir_commit ];then
+  chmod 777 $base_dir_commit
+  if [ -d $base_dir_commit/fix_sfc ]; then
+    chmod 777 $base_dir_commit/fix_sfc
+  fi
+  rm -fr $base_dir_commit
+fi
+
+mkdir -p $base_dir_commit
+
+for files in *.nc
+do
```

Comment:
I see there's no 'cd' here before NetCDF files are copied. Is the `reg_tests` directory defined to be the location of all the RT NetCDF files somewhere?  Will this work for for all utilities?

Reply:
This script is run from the test's working directory. So the files are copied from the working directory to the baseline directory. The baseline directory for the particular test is stored in `base_dir.`  And `base_dir` is a function of `HOMEreg`, which is the regression test root directory and is passed in from the driver script.